api/bases/keystone.openstack.org_keystoneapis.yaml

```diff
@@ -84,6 +84,19 @@ spec:
                   files. Those get added to the service config dir in /etc/<service>
                   . TODO: -> implement'
                 type: object
+              enableFederation:
+                default: false
+                description: Enablement of Federation configuration
+                type: boolean
+              enableInsecureDebug:
```

Comment:
enableInsecureDebug and enableKeystoneDebug  feel like settings that should be set in custom.conf.  I would imagine that in production, they would always be set to false.  This is only useful in a dev/debugging environment.  If the custom.conf setting is not working, thats a different problem.

Reply:
Done